lib/services/role.go

```diff
@@ -792,8 +870,11 @@ func (set RuleSet) Match(whereParser predicate.Parser, actionsParser predicate.P
 	if len(set) == 0 {
 		return false, nil
 	}
-	// check for wildcard resource matcher
-	for _, rule := range set[Wildcard] {
+
+	// check for matching resource by name
```

Comment:
What's the problem you are solving with the new priorities?

Reply:
explained in comments, take a look